src/main/java/ch/njol/skript/effects/EffKill.java

```diff
@@ -73,11 +75,14 @@ protected void execute(final Event e) {
 				entity = ((EnderDragonPart) entity).getParent();
 			}
 
+			if (IGNORE_TOTEM) {
+				HealthUtils.setHealth((Damageable) entity, 0);
```

Comment:
missing return? or else you set health to 0 and then do damage, or intended?

Reply:
why was this resolved?

Reply:
I lowkey don't know why, I must of forgotten to push changes the first time... oops